src/components/Select/Select.stories.jsx

```diff
@@ -50,3 +50,9 @@ NoBlankOption.args = {
   ...defaultArgs,
   includeBlankOption: false,
 };
+
+export const AriaLiveRegion = Template.bind({});
+AriaLiveRegion.args = {
+  ...defaultArgs,
+  ariaLiveRegionText: 'The following branch was selected: ',
+};
```

Comment:
I am not opinionated about Prop Names, if y'all have something different, let me know!

Reply:
This looks great! It's very clear what this is doing. :+1: